content/PHPDataObjects/DataObjects.q.md

```diff
@@ -848,6 +952,119 @@ Correct the code so that it outputs the strings `Successfully connected to the d
 ```
 
 
+:::
+
+/// type=REPL, readonly=true, filename=[connection.php,student.php], init=[commands/DataObjects/UpdateBirthDateJohnSmith.sql]
```

Comment:
Which one should come first? Is it `filename` or `init`? Please check. I think we need to be consistent with the sequence. Thanks.